added path example for windows users (#15365)
DataDog/documentation: content/en/agent/logs/_index.md

    source: "<SOURCE>"
```

> If you are using Windows use `path: "<DRIVE_LETTER>:\\<PATH_LOG_FILE>\\<LOG_FILE_NAME>.log"` for the path. Also verify that the `ddagentuser` has all the read/write access to the log file.

Reviewer: ```suggestion
On Windows, use `path: "<DRIVE_LETTER>:\\<PATH_LOG_FILE>\\<LOG_FILE_NAME>.log"` for the path, and verify that the user `ddagentuser` has read and write access to the log file.
```
